### PR TITLE
Bug 1350467 - Add method Bugzilla->preload_features() to be called in mod_perl.pl startup

### PR DESCRIPTION
**Pull request by dylanwh**

### CODE REVIEW

File: Bugzilla/CPAN.pm [hunk 1]
```diff
@@ -104,6 +104,16 @@ sub feature {
     return $FEATURE_LOADED{$feature_name} = 1;
 }
 
+sub preload_features {
+    my ($class) = @_;
+    return 0 unless _CAN_HAS_FEATURE;
```
Comment (floatingatoll) [inline Bugzilla/CPAN.pm]: Does it offer any material performance benefit to move this line above the my()@_ invocation?

File: Bugzilla/CPAN.pm [hunk 2]
```diff
@@ -104,6 +104,16 @@ sub feature {
...
+    my $meta = $class->cpan_meta;
+
+    foreach my $feature ($meta->features) {
+        $class->feature($feature->identifier);
+    }
```
Comment (floatingatoll) [inline Bugzilla/CPAN.pm]: This is more concise and with two less my() invocations, but the usual concerns about map() readability apply. ```map { $class->feature($_->identifier) } $class->cpan_meta->features;``` might work out.

Comment (dylanwh) [inline Bugzilla/CPAN.pm]: map in void context is a thing I try to never do, it's bad style
(although perl is smart enough to throw the return values away)

File: Bugzilla/CPAN.pm (hunk 1 above)
Comment (dylanwh) [inline Bugzilla/CPAN.pm]: pretty much I've never been able to measure a difference from using variables or not, it seems like just method calls / function calls are slow.

**Pull request merged**